route/config/route_config_params.yaml

```diff
@@ -4,7 +4,7 @@ max_crosstrack_error: 2.0
 
 # Double: downtrack_range
 # Units: meter
-destination_downtrack_range: 0
+destination_downtrack_range: 10
```

Comment:
What does this parameter do and why was it changed?

Reply:
This parameter tracks how far from the end of the route can we stop to get the route_completed message.
It was changed because we are stopping slightly before the end point of route

Reply:
Ok. Please update the description then, so the use of the parameter is clearer.